### ISSUE TITLE
Add local dependencies for sails, ember-cli and sane-cli

### ISSUE THREAD

**Issue by Globegitter** [sane/sane#16]
This seems to be the preferred way of doing things that could simplify the install to `npm i -g sane-cli` (if docker is not used), but would add an additional package.json

The other alternative would be to use the `postinstall` hook, having a script that checks for `sails` and `ember-cli` to be present, otherwise it instlalls it.

**Comment by Globegitter** [comment]
Mostly implemented as of version 0.0.19

**Comment by IanVS** [comment]
What still needs to be done here?

**Comment by Globegitter** [comment]
The global sails is still being used if you don't use docker. Haven't really made up my mind yet we want to keep that behaviour or have local sails in that case as well.

**Comment by IanVS** [comment]
Personally, I really like how sane-cli asks which ember to use if the global version is different from the local. Could something similar be done for sails?

**Comment by Globegitter** [comment]
Yeah, one problem I faced is, it actually adds some noticeable loading time the check. So another one would add another noticeable loading time. That was really what made me not implement it so far. But yeah that is probably the way forward :)
